source/refunding_payments/index.html.md.erb

```diff
@@ -79,54 +78,54 @@ with a previous refund of £30:
   },
 ```
 
-## Specifying the expected refund available
+## Find out the expected refund available
 
-When you submit a refund request for a payment via the API, you can optionally
-specify a `refund_amount_available` value in the body of the request. This
-is so you can provide the total amount of the original payment you expect to
-be available for refund at the time your refund request is made.
+When you submit a refund request via the API, you can optionally include a
+`refund_amount_available` value in the request body. This is so you can
+provide the total amount of the original payment you expect to be available
+for refund, at the time your refund request is made.
 
 The purpose of this is to prevent accidentally processing a partial refund
 more than once, by rejecting requests where your `refund_amount_available`
-does not match the real amount that’s available to be refunded.
+does not match the real amount that’s available for refund.
 
-For example, a payment is made for £5, but later the user requires a £2
+For example, a user makes a payment of £5, and later the user requires a £2
 refund. Your system for processing refunds submits a request for a £2 refund
-to the GOV.UK Pay API, but it accidentally gets sent twice. Without a
-`refund_amount_available` specified, GOV.UK Pay would have no way to tell
-the second request was a mistake. It would process both requests,
-generating two refunds of £2 each.
-
-In the same scenario, if you had specified the ``refund_amount_available`` as
-£5. The first request still succeeds, leaving £3 available to be refunded.
-When the accidental duplicate request comes in, it has a
-``refund_amount_available`` of £5, even though only £3 is available, so GOV.UK
-Pay can tell that it’s a stale request, and it is rejected.
-
-We recommend that your service tracks the expected refund amount available and
-submits a ``refund_amount_available`` value whenever you request a refund via
+to the GOV.UK Pay API, but the request accidentally gets sent twice. Without
+finding out `refund_amount_available`, GOV.UK Pay would have no way to tell
+the second request was a mistake. It would process both requests, generating
+two refunds of £2 each.
+
+As another example, you may have specified the `refund_amount_available` as
+£5. The first refund request for £2 still succeeds, leaving £3 available for
+refund. When the accidental second request arrives, the payment has a
+``refund_amount_available`` of £5 even though only £3 is available. GOV.UK
+Pay determines that it’s a stale request, and rejects the refund request.
+
+Your service should track the expected refund amount available. It should
+return a `refund_amount_available` value whenever you request a refund via
```

Comment:
Not sure if I'm going mad but I'm confused about whether this particular line should reference `refund_amount_available` or `amount_available`

Reply:
refund_amount_available forms part of the request body when a refund is submitted via API

Reply:
In which case presumably this should be `amount_available`?

Reply:
looks good